src/main/java/de/codefor/le/crawler/LvzPoliceTickerDetailViewCrawler.java

```diff
@@ -181,11 +176,8 @@ static Date extractDate(final String date) {
 
     private static void extractArticle(final Document doc, final PoliceTicker dm) {
         final var content = "articlecontent";
-        var cssQuery = ".pdb-article-body > .pdb-richtext-field > p";
```

Comment:
This block can be removed. If a article is not a paid one, teaser and article can be simple found. If the article is a paid one, the teaser will be found too but the article is not visible, so articlecontent will be emty.